kessel/grpc/grpc.go

```diff
@@ -0,0 +1,31 @@
+package grpc
+
+import (
+	"context"
+	"fmt"
+	"github.com/project-kessel/kessel-sdk-go/kessel/auth"
```

Comment:
is this breaking the rule that we have defined in the spec for grpc, i.e not import anything from peer packages. Is auth a peer package? But unsure of how we would do this without importing the auth package.

https://project-kessel.github.io/docs/contributing/client-api/grpc/

Reply:
That's indeed a conflict with the spec - I think the same happens in others sdks.

I guess we should allow this import (and update the spec)

Reply:
As discussed, we will allow the auth import since it is a generic peer package. The rule in the spec is referring to imports in the other direction, like auth importing grpc for ex.